app/grants/templates/grants/index.html

```diff
@@ -85,11 +95,53 @@ <h1 class="font-title-xl font-weight-semibold">{% trans 'Provide sustainable fun
                 {% trans 'Become a Matching Partner' %}
               </a>
             {% else %}
-              <div class="btn btn-gc-blue font-body font-weight-semibold" onclick="processPayment()">
-                {% trans 'Become a Matching Partner for Round 4' %}
+              <div class="btn btn-gc-blue font-subtitle font-weight-semibold" onclick="processPayment()">
+                {% trans 'Become a Matching Partner' %}
               </div>
             {% endif %}
           </div>
+          <div class="col-md-6">
```

Comment:
Do we need this ?
The sponsors are already displayed here already 
![image](https://user-images.githubusercontent.com/5358146/71807707-ef376e80-3091-11ea-8007-258b27439eda.png)